### DISCUSSION TITLE
Mismatch in documentation / "mobx.es6.js" for v6

### DISCUSSION THREAD

**Discussion by peabnuts123** [mobxjs/mobx#2880] [General]
I may have missed something but all signs still point to aliasing mobx to `node_modules/mobx/lib/mobx.es6.js` in the documentation. However, the version I have installed (6.1.8) does not have this file; instead it has `src` and `dist` folders. I assume this is a v6 change, but I'm not sure what the guidance is here. Has ES6 become the default target? Nobody else in the GitHub issues seems to be confused by this, so I am just looking for guidance / perhaps the documentation needs to be updated.

**Comment by mweststrate** [comment]
You can find the different builds that are available here: https://unpkg.com/browse/mobx@6.1.8/dist/. Feel free to open a PR on the docs where you spotted the problem

**Reply by peabnuts123** [reply]
@mweststrate How is one supposed to include mobx nowadays? Is there still a "ES6-only" build? Do you need an alias at all? That's what I'd like to know.

**Reply by mweststrate** [reply]
Sorry I'm missing all context here. The normal way of including MobX is adding it to your package.json, `import stuff from "mobx"` and let your bundler sort all the rest. It can be that for some very special case you need an alias, but then please elaborate on your question :)

**Reply by peabnuts123** [reply]
Okay, weird. Looks like there are actually 2 websites for mobx. [This website](https://www.mobxjs.com/README.html) is still saying:
> Tip: the main entry point of the MobX 5 package ships with ES5 code for backward compatibility with all build tools. But since MobX 5 runs only on modern browsers anyway, consider using the faster and smaller ES6 build: lib/mobx.es6.js. For example by setting up a webpack alias: resolve: { alias: { mobx: __dirname + "/node_modules/mobx/lib/mobx.es6.js" }}

But [this website](https://mobx.js.org/README.html) seems to have updated. I am assuming that v6 has since updated and no-longer needs this. But the old v5 website seems to still exist.

**Reply by mweststrate** [reply]
wow that's weird, that is not the official website indeed, never saw that before and contains an old version

**Reply by mweststrate** [reply]
mobx.js.org is the official one for all clarity

**Reply by peabnuts123** [reply]
I think my question has been answered, there is no need for an ES6-specific build any more. Perhaps you can find out who owns the rogue site and get that taken down? That should limit any confusion.
